ada-project-docs/wave_05.md

```diff
@@ -175,13 +148,19 @@ As a client, I want to be able to make a `DELETE` request to `/goals/1` when the
 }
 ```
 
-### Delete Goal: No Matching Goal
+### No matching Goal: Get, Update, and Delete
+
+As a client, if I make any of the following requests:
+
+  * `GET` `/goals/<goal_id>`
+  * `UPDATE` `/goals/<goal_id>`
+  * `DELETE` `/goals/<goal_id>`
 
-As a client, I want to be able to make a `DELETE` request to `/goals/1` when there are no matching goals and get this response:
+and there is no existing goal with `goal_id`
 
-`404 Not Found`
+The response code should be `404`.
 
-No response body.
+You may chose the response body.
```

Comment:
Small typo. I might also add the line about completing the test to each instance where it's needed for consistency. 

```suggestion
You may choose the response body.

 Make sure to complete the tests for non-existing tasks to check that the correct response body is returned.
```